gcs/src/it/scala/org/broadinstitute/monster/storage/gcs/GcsApiIntegrationSpec.scala

```diff
@@ -79,8 +84,9 @@ class GcsApiIntegrationSpec
     val blob = BlobId.of(bucket, blobPath)
     val blobInfo = BlobInfo.newBuilder(blob).setContentType("text/plain").build()
 
+    // TODO best way to get bytes?
```

Comment:
I would have `writeTestFile` (and `writeGzippedTestFile`) take a `Chunk[Byte]` as an argument. Then `setup`can use that chunk.